src/Plots.jl

```diff
@@ -35,6 +35,7 @@ using Base.Meta
 using Requires
 using Reexport
 using Unzip
+@reexport using Preferences
```

Comment:
```suggestion
using Preferences
```

I'd prefer not exporting this.

Reply:
So you'd recommend users to call `Plots.Preferences.set_preference!(Plots, "default_backend" => "plotly")`?
I thought that is a bit long

Reply:
I would prefer writing our own `PlotsPreferences` module, define our backend selection helpers / functions (with error checks), and export only what's necessary from that module (leaking into `Plots`).

See for example the `GRPreferences` module:
https://github.com/jheinen/GR.jl/blob/master/src/preferences.jl#L97-L118.